typescript/infra/scripts/check/check-warp-deploy.ts

```diff
@@ -25,7 +25,7 @@ async function main() {
   const failedWarpRoutesChecks: string[] = [];
 
   // TODO: consider retrying this if check throws an error
-  for (const warpRouteId of Object.values(WarpRouteIds)) {
+  for (const warpRouteId of Object.values(CheckerWarpRouteIds)) {
```

Comment:
extending my previous comment, if the only use of the new enum is for this change, we could do `Object.keys(warpConfigGetterMap)`

Reply:
I'll do this instead.

We access `WarpRouteIds` here https://github.com/hyperlane-xyz/hyperlane-monorepo/blob/389fd91ec6f00023ee4f1b71f85e0eb9ebd5c4d9/typescript/infra/config/environments/mainnet3/agent.ts#L358

And we don't support `EthereumEclipseUSDC` yet in the checker